### PR TITLE
Enable claim reward when compounding is on.

### PR DESCRIPTION
**Pull request by bobo-k2**
**Pull Request Summary**

Claim reward button is visible no matter of compounding status.
image

If user claim rewards when compounding is on, rewards are added to balance and restaked.
If user claim rewards when compounding is off, rewards are only added to balance.

@Dinonard  please check the above statement

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

**This pull request makes the following changes:**

**Changes**
- claim button visibility doesn't depend on compounding status.

### CODE REVIEW

**Comment by Dinonard** [comment]
@bobo-k2
I'd rephrase it to:
If user claim rewards when compounding is on, rewards are added to balance and restaked.
If user claim rewards when compounding is off, rewards are only added to balance.

**Review by hoonsubin — APPROVED** [review]
LGTM

**Review by impelcrypto — APPROVED** [review]
